README.md

```diff
@@ -1 +1,283 @@
+        { "27": 170 },
+        { "28": 260 },
+        { "29": 230 },
+        { "30": 210 },
+        { "31": 180 }
+    ]
+}
+
+```
+
+response:
+```json
+GET /orderedItemsByPeriod?period=month&year=2023
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "period": "month",
+    "year": "2023",
+    "orderedItems": [
+        { "january": 6528 },
+        { "february": 7324 },
+        { "march": 8452 },
+        { "april": 7890 },
+        { "may": 9103 },
+        { "june": 8675 },
+        { "july": 9234 },
+        { "august": 8820 },
+        { "september": 9345 },
+        { "october": 8901 },
+        { "november": 8123 },
+        { "december": 9576 }
+    ]
+}
+```
+
+#### 4. Get leftovers
+`GET /getLeftOvers?sortBy={value}&page={page}&pageSize={pageSize}`: Returns the inventory leftovers in the coffee shop, including sorting and pagination options.
+##### **Parameters**:
+- `sortBy` _(optional)_: Determines the sorting method. Can be either:
+    - `price`: Sort by item price.
+    - `quantity`: Sort by item quantity.
+- `page` _(optional)_: Current page number, starting from 1.
+- `pageSize` _(optional)_: Number of items per page. Default value: `10`.
+##### **Response**:
+- Includes:
+    - A list of leftovers sorted and paginated.
+    - `currentPage`: The current page number.
+    - `hasNextPage`: Boolean indicating whether there is a next page.
+    - `totalPages`: Total number of pages.
+
+response example:
+```json
+GET /getLeftOvers?sortBy=quantity?page=1&pageSize=4
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "currentPage": 1,
+    "hasNextPage": true,
+    "pageSize": 4,
+    "totalPages": 10,
+    "data": [
+        {
+            "name": "croissant",
+            "quantity": 109,
+            "price": 950
+        },
+        {
+            "name": "sugar",
+            "quantity": 93,
+            "price": 50
+        },
+        {
+            "name": "muffin",
+            "quantity": 63,
+            "price": 350
+        },
+        {
+            "name": "milk",
+            "quantity": 1,
+            "price": 200
+        }
+    ]
+}
+```
+
+
+Since the auditor will not have access to your local database and its data, you need to create a migration file named `init.sql`. This file should include:
+
+1. Scripts to Create All Required Tables
+    Define the structure of the tables (`user`, `order`, `menu`, `menu_item`, `inventory`, `ingredient`, etc.) in SQL, including column definitions, data types, and constraints.
+
+2. Definition of Relationships Between Tables
+    Ensure that relationships such as `FOREIGN KEY` constraints are properly defined.
+3. Mock Data
+	Include sample data for testing purposes:
+	- Test users
+	- Sample orders
+	- Menu items
+	- Inventory records
+	- Ingredients
+
+## Guidelines from Author
+
+In the initial stages of working with tables and relationships, it can be challenging to understand and organize everything. Therefore, it is recommended to use **pgAdmin**
```

Comment:
you could use first person instead of the third person to be to make guideline more personal and less formal